resources/moneropedia/miners.md

```diff
@@ -7,14 +7,12 @@ summary: "the process of cryptographically computing a mathematical proof for a
 
```

Comment:
Can we rename this file to `mining.md` to match the Entry name?

Reply:
Makes sense. Renamed.